var/spack/repos/builtin/packages/netlib-lapack/package.py

```diff
@@ -98,9 +98,14 @@ def blas_libs(self):
         }
         key = tuple(sorted(query_parameters))
         libraries = query2libraries[key]
-        return find_libraries(
-            libraries, root=self.prefix, shared=shared, recursive=True
-        )
+        prefix = self.prefix
+        search_paths = [(prefix.lib, False), (prefix.lib64, False), (prefix, True)]
+        for path, recursive in search_paths:
+            libs = find_libraries(
+                name, root=path, shared=shared, recursive=recursive
```

Comment:
There's some flake errors but I think they are indicating you should be using `libraries` here. Ideally we could wrap this logic up and put it in a function in `llnl.util.filesystem` and then reuse it across `Spec._libs_default_handler` and this function. But if you're crunched for time could you add a TODO for that here?

Reply:
yep, I caught another error too.  The latest update has a potential fix/abstraction